md/BonitaBPM_platform_setup.md

```diff
@@ -31,33 +31,40 @@ It is composed of the following items:
 
 ### Functions
 
-This tool has 3 functions:
+This tool has 4 functions:
 
 * Initialize the database:`init`
-It creates the database schema and pushes the initial configuration in the database.   
+
+It creates the database schema and pushes the initial configuration in the database.
```

Comment:
does the lines following the bullet points need to be aligned with the bullet points?